k8s/prod/common/ethos/repl-docmosis-service.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: repl-docmosis-service
+  namespace: ethos
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: repl-docmosis-service
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ethos-repl-docmosis-backend
+    version: 0.1.4
+  values:
+    java:
+      replicas: 2
+      memoryRequests: '512Mi'
+      cpuRequests: '250m'
+      memoryLimits: '2048Mi'
+      cpuLimits: '1500m'
+      readinessDelay: 45
+      readinessTimeout: 5
+      readinessPeriod: 15
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "162CD82F0363F36E81218AD218C3C19F862CD480"
+      image: hmctspublic.azurecr.io/ethos/repl-docmosis-backend:prod-ec0fade0
+      ingressHost: ethos-repl-docmosis-backend-prod.service.core-compute-prod.internal
+      environment:
```

Comment:
It looks like these 2 values need overriding for production:
https://github.com/hmcts/ethos-repl-docmosis-service/blob/master/charts/ethos-repl-docmosis-backend/values.yaml#L23-L24

Reply:
Added